### PR TITLE
community: Nicolas Harrand becomes integrator

### PR DESCRIPTION
**Pull request by monperrus**
Dear all,

I'd like to nominate Nicolas Harrand (@nharrand) to become an integrator in Spoon. Nicolas has done several key contributions over the past year:

* he has added support for paths in order to uniquely identify AST nodes (#1874), this feature is unique (to my knowledge, it does not exist in any other source code analysis library for Java), and enables original source code analysis over time and versions.
* he has added support for analysis of binary files with decompilation (#2455)
* he has laid down the foundations of a modularized Spoon (#2766), which will likely be key in the future
* he has ported Spoon to JDK 11 (#2789). This kind of infrastructure contribution is also very important in the long term.

In addition, Nicolas has proven his ability to communicate constructively and respectfully over issues and pull-requests.

What do you think?

### CODE REVIEW

**Comment by surli** [comment]
+1

**Comment by pvojtechovsky** [comment]
Very good idea! I agree 👍

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ How to become integrator? The integrators are the developers who have made signi
 
 Current integrators (alphabetical order):
 
+- Nicolas Harrand [@nharrand](https://github.com/nharrand/)
```
Comment (pvojtechovsky) [inline CONTRIBUTING.md]: It is just not in alphabetical order ;-)

Comment (monperrus) [reply CONTRIBUTING.md]: fixed :-)

**Comment by monperrus** [comment]
6 days, nobody objects, ready for merge?

**Comment by monperrus** [comment]
A round of applause for Nicolas @nharrand!
https://github.com/INRIA/spoon/invitations

**Comment by surli** [comment]
:clap:

**Comment by nharrand** [comment]
Thanks all! I'll try to be worthy :innocent:.

**Comment by pvojtechovsky** [comment]
👏

**Pull request merged**